docs/docs/configuration/overview.md

```diff
@@ -108,6 +108,7 @@ An example [oauth2-proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/
 | `--redis-sentinel-connection-urls` | string \| list | List of Redis sentinel connection URLs (e.g. `redis://HOST[:PORT]`). Used in conjunction with `--redis-use-sentinel` | |
 | `--redis-use-cluster` | bool | Connect to redis cluster. Must set `--redis-cluster-connection-urls` to use this feature | false |
 | `--redis-use-sentinel` | bool | Connect to redis via sentinels. Must set `--redis-sentinel-master-name` and `--redis-sentinel-connection-urls` to use this feature | false |
+| `--request-id-header` | string | Request header to use as the request ID in logging | X-Request-Id |
```

Comment:
I know headers aren't case sensitive, but, something in me is saying this should be:
```suggestion
| `--request-id-header` | string | Request header to use as the request ID in logging | X-Request-ID |
```
Very much a linter nit ;P

Reply:
I had it like that the first time 😄 

It made unit tests annoying to just keep using the const because of the under the hood call to `textproto.CanonicalMIMEHeaderKey`.

Want me to try to flip it back? I've seen both forms documented `X-Request-ID vs `X-Request-Id`

Reply:
Which version does CanonicalMIMEHeaderKey spit out? Let's be consistent with that everywhere (which I think based on your comment is `X-Request-Id`?)

Reply:
Yeah, that's correct - it will convert it to `X-Request-Id` -- let me check if I left any `X-Request-ID` lingering maybe in docs.